import/webview/WebView.qml

```diff
@@ -31,19 +31,31 @@ RawWebView {
         webview.loadFrameScript("chrome://embedlite/content/embedhelper.js");
         webview.loadFrameScript("chrome://embedlite/content/SelectAsyncHelper.js");
         webview.addMessageListeners([
-            "embed:linkclicked",
-            "embed:filepicker",
-            "embed:selectasync",
-            "embed:alert",
-            "embed:confirm",
-            "embed:prompt",
-            "embed:auth",
-            "embed:permissions",
-            "embed:login",
-            "Content:ContextMenu"]);
+                                        "embed:linkclicked",
+                                    ]);
+    }
+
+    PickerOpener {
+        id: pickerOpener
+
+        property QtObject pageStackOwner: helper.findParentWithProperty(webView, "pageStack")
```

Comment:
Not needed. Moved logging from PickerOpener Component.onCompleted to message handler. So, we do find pageStack but pagestack is unknown when PickerOpener's Component.onCompleted is called.